### ISSUE TITLE
Implement Landing Page

### ISSUE THREAD

**Issue by theBeginner86** [layer5io/docs#12]
Implement Landing page

Refer Figma Design: https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?type=design&node-id=22103%3A13238&mode=design&t=viNBpM3L39ZwaNgB-1

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by leecalcote** [comment]
Mostly done.

**Comment by leecalcote** [comment]
GOAL:


![docs layer5 io](https://github.com/layer5io/docs/assets/7570704/94088415-a997-4bcb-b634-e9dfc497b0fb)

**Comment by Aniumbott** [comment]
@leecalcote can I work on this?

**Comment by leecalcote** [comment]
Thanks, @Aniumbott

**Comment by Aniumbott** [comment]
Should I work on a different branch to avoid the conflicts as the current one is live and it might happen that it will take some time to refine the current one to the GOAL?

**Comment by leecalcote** [comment]
@Aniumbott, yes, each time that you contribute whether with a bug fix or an enhancement or a chore or any change, a separate branch is the way to go. 👍

**Comment by Aniumbott** [comment]
I completed the navbar.
The updated files can be found [here](https://github.com/Aniumbott/docs/tree/feat/navigation).

https://github.com/layer5io/docs/assets/76243585/f9fa765a-fa43-494f-ad36-2efd30bfb801

Should I open a PR for the master branch?

**Comment by iArchitSharma** [comment]
@leecalcote so we are removing cloud and meshmap link from navbar?

**Comment by leecalcote** [comment]
No. Much has happened to the site since the design was first mocked up. Please refractor as appropriate.

**Comment by Aniumbott** [comment]
> Its been a while since any updates were conveyed regarding progress. @Aniumbott Are you still working on this issue?

I'm having exams you can take over if you want.

**Comment by psankhe28** [comment]
@sandramsc @leecalcote Can I work on this issue?

**Comment by iArchitSharma** [comment]
@psankhe28 Thanks for your interest in helping out! We've recently made some updates to the Landing page. However, there are plenty of other areas where you can contribute.
I suggest checking out the list of open issues here: https://github.com/layer5io/layer5/issues

**Comment by leecalcote** [comment]
Thank you, @iArchitSharma
